db/fixtures/notification_types.yml

```diff
@@ -113,4 +113,9 @@
   :message: 'The operation %{op_name} %{op_arg} on %{mw_server} has failed to complete. Please check the logs for further details.'
   :expires_in: 24.hours
   :level: :warning
-  :audience: global
\ No newline at end of file
+  :audience: global
+- :name: vm_snapshot_failure
+  :message: 'Failed to %{snapshot_op} snapshot of %{subject}: %{error}'
+  :expires_in: 24.hours
+  :level: :warning
+  :audience: global
```

Comment:
TODO: I really want this to be `:audience: user` I just need to get the `:user_id` down to the snapshot call